weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewerSection.tsx

```diff
@@ -153,17 +153,22 @@ const ObjectViewerSectionNonEmpty = ({
     setMode('collapsed');
     setExpandedIds([]);
   };
+
+  const isExpandAllSmall =
+    !!apiRef?.current?.getAllRowIds &&
+    getGroupIds().length < EXPANDED_IDS_LENGTH;
```

Comment:
Would it make sense for this to be something like `getGroupIds().length - expandedIds.length < EXPANDED_IDS_LENGTH`? i.e. if you've expanded 990 / 1000 already the message would be "Expand all" instead of "Expand next 200 rows"

Reply:
yah that makes sense to me